### ISSUE TITLE
About NExSS: PIs and Current Teams need to be updated to reflect who currently has NASA funding.  We will have to send you (or Alan?) the relevant info, and reorganize that page into Current Steering Council, and Alumnae.

### ISSUE THREAD

**Issue by andyburnett** [Knowinnovation/nexss#11]
Jessica, or Alan can send us this information

**Comment by jlnoviello** [comment]
I'll have to get this list from Dan Larsen before he leaves

**Comment by jlnoviello** [comment]
Andy, Dan Larsen has compiled this information into a set of Google spreadsheets. I can add you to the file. Whose emails should I send the invitation to?

**Comment by andyburnett** [comment]
Please add andy.burnett@knowinnovation.com for now.

**Comment by jlnoviello** [comment]
Shared! There's a lot of information on it but I think the PI name and the title of the project are the most important to get onto the website first. I can add on more information later once we're done collecting it from the teams.

**Comment by andyburnett** [comment]
@jlnoviello Hmm, I didn't get an alert for the new document. Could you email me the link please?

**Comment by jlnoviello** [comment]
I forgot about this comment, sorry Andy! Here is the direct link: 
https://docs.google.com/spreadsheets/d/19q4ZuXZT5yV7AUip8SzJOMetg_TA47ckJ0WvtSUNRg4/edit?pli=1#gid=0

Dan Larsen had also started gathering information from the new PIs to accompany their team / PI profile on the website, which I don't think has been implemented yet (at least on the old website).  You can find the data that he gathered from the PIs here:  https://docs.google.com/spreadsheets/d/1Vr8G8ls8ijLCM-hbKVHayQxaqGOawPzyfqkGAZhqA24/edit?usp=drive_web&ouid=117724720678333384355.

**Comment by andyburnett** [comment]
@jlnoviello I just noticed that the research interests data in this file (https://docs.google.com/spreadsheets/d/19q4ZuXZT5yV7AUip8SzJOMetg_TA47ckJ0WvtSUNRg4/edit?pli=1#gid=0) is not consistent with the PI research interest data in this file (https://docs.google.com/spreadsheets/d/1Vr8G8ls8ijLCM-hbKVHayQxaqGOawPzyfqkGAZhqA24/edit?usp=drive_web&ouid=117724720678333384355)

**Comment by jlnoviello** [comment]
I think that it's fine to include the PI information and affiliation, the title of the work, and the website (if available) for now. If a team has research interests listed then we can include those too, perhaps as keywords under the other information, but I think it will be my future job to link these teams up with each other based on research focus. We do need to be careful about some people who are now no longer the PIs of current NExSS teams because their funding has expired (Steve Desch is one such person). They will need to move to the NExSS Alumni section towards the bottom of the page (at least I think that's what it's called and that's where it was put? We can discuss that on Friday).

Long story short: use the information that's there now and I'll get the rest later if we decide we need it.

**Comment by jlnoviello** [comment]
I think we have all the information we're going to get for now, so I'll close this issue and we can reopen if we need to
